gradle-plugin/src/main/kotlin/io/reflekt/plugin/util/Util.kt

```diff
@@ -0,0 +1,20 @@
+package io.reflekt.plugin.util
+
+import org.gradle.api.Project
+import org.gradle.api.file.SourceDirectorySet
+import org.gradle.api.internal.HasConvention
+import org.gradle.api.tasks.SourceSet
+import org.gradle.api.tasks.SourceSetContainer
+import org.jetbrains.kotlin.gradle.plugin.KotlinSourceSet
+
+internal inline fun <reified T : Any> Project.myExtByName(name: String): T = extensions.getByName(name) as T
```

Comment:
that's the first time we use 'my' in identifiers, I guess, let's stick to the project style?

Reply:
I am not sure that I should it fix now since we will delete all these functions during rewriting this file (delete deprecated HasConvention)